docs/manual/en/introduction/How-to-create-VR-content.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<base href="../../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+
+<body>
+	<h1>[name]</h1>
+	<br />
+
+	<p>
+		This guide provides a brief overview of the basic components of a web-based VR application
+		made with three.js.
+	</p>
+
+	<h2>Workflow</h2>
+
+	<p>
+		You have to include [link:https://github.com/mrdoob/three.js/blob/master/examples/js/vr/WebVR.js WebVR.js]
+		into your project. *WEBVR.createButton()* does two important things: First, it creates a button which indicates
+		VR compatibility. Second, it initiates a VR session if the user activates the button. The only thing you have
+		to do is to add the following line of code to your app.
+	</p>
+
+	<code>
+document.body.appendChild( WEBVR.createButton( renderer ) );
+	</code>
+
+	<p>
+		Next, you have to tell your instance of *WebGLRenderer* to enable VR rendering:
+	</p>
+
+	<code>
+renderer.vr.enabled = true;
+	</code>
```

Comment:
Off topic: Do we still need `renderer.vr.enabled` property? I haven't read though the VR related code in `WebGLRenderer` but is there any problems if it's always `true`?

Reply:
`WebGLRenderer` would execute code like this if the flag is always to `true`:

https://github.com/mrdoob/three.js/blob/3e5b0f2727792cd200b177c42a5750322065bf31/src/renderers/WebGLRenderer.js#L1148-L1152

I don't think non-VR applications should go through this code path.

Reply:
What I'm thinking right now is.... If `renderer.vr.enabled` is always `true` or is `true` as default, and we recommend to always use `renderer.setAnimationLoop` instead of `requestAnimationFrame`, what users need to do for creating VR contents from the non-VR ones is just adding "Enter VR" button. It'd be more easier.

Reply:
About `vr.submitFrame()`, it does do nothing in non-VR mode.

Reply:
> About vr.submitFrame(), it does do nothing in non-VR mode.

Yes, you're right. I just wanted to highlight with my previous post that it might be confusing for the reader if `vr.submitFrame();` is called although VR is not used at all^^.

>  If renderer.vr.enabled is always true or is true as default,

This would break all existing code which uses an ordinary animation loop. I mean something like
```js
function animate() {

    requestAnimationFrame( animate );
    renderer.render( scene, camera );

}
```
Not sure if we want to risk this...

Reply:
BTW, my comments here doesn't block merging this PR. I was just curious.